code/app/src/main/java/com/example/goldencarrot/views/AdminAllUsersView.java

```diff
@@ -74,21 +74,21 @@ public void onItemClick(AdapterView<?> adapterView, View view, int position, lon
             }
         });
     }
-    public void getUsersFromFirestore(List<DocumentSnapshot> listOfUsers) {
-        // convert all documents into users
-        for (int i = 0; i < listOfUsers.size(); i++) {
-            try {
-                DocumentSnapshot userFromDb = listOfUsers.get(i);
-                User newUser = new UserImpl(userFromDb.getString("email"),
-                        userFromDb.getString("userType"),
-                        userFromDb.getString("username"));
-                // add user to user data list
-                dataUserList.add(newUser);
-                Log.i(TAG, "Successfully added " + userFromDb.getString("username"));
-            } catch (Exception e) {
-                Log.e(TAG, "Invalid user type, user not added");
-            }
-        }
-    }
+//    public void getUsersFromFirestore(List<DocumentSnapshot> listOfUsers) {
```

Comment:
This method needs some revision as it uses the username which is deprecated. Changed to name